Header header
std_msgs/String name
std_msgs/Bool run
std_msgs/Bool shut_down